tests/PrecisionTest.php

```diff
@@ -45,7 +45,7 @@ public function roundingDates(): array
             [Precision::DAY, '2018-02-05 00:00:00', '2018-03-05 00:00:00'],
             [Precision::HOUR, '2018-02-05 11:00:00', '2018-03-05 11:00:00'],
             [Precision::MINUTE, '2018-02-05 11:11:00', '2018-03-05 11:11:00'],
-            [Precision::SECOND, '2018-02-05 11:11:11', '2018-03-05 11:11:11'],
```

Comment:
Could you undo these styling changes?

Reply:
Rolled back last commit to undo changes.